parachain/src/chain_spec.rs

```diff
@@ -356,7 +359,201 @@ fn testnet_genesis(
+                    // 5CzNqQVuHu8EbtXryz8kcnBvrLnaKbPoFaFUgmRfvicCYB8Y
+                    hex!["28ef3806091fcb50df12003d4c62c81cdd28d2e152ea5bf01189d8cd8ecb0927"].unchecked_into(),
+                    // 5GN6a5uU6BNLTUZYYBMHDAj3UHnVbQcs5Hhoizi6K2FbnhbV
+                    hex!["be34916ea4a7175d211cf041550346800d798d88208b1cf507025a6ebed12247"].unchecked_into(),
+                    // 5HGvZ1WCyYarfPdws7qg6qdFY6i6jKEvUugtp3vmhCBvbcdY
+                    hex!["e67ed95d635b20c401be1ea8d682d28a09e1d32304cdaf1d87c47904d6f1bc38"].unchecked_into(),
+                    // 5E5LuHWr4Niv46MFfApqyf88Y1G9QRLTpVuSbDSRTUKoKwzm
+                    hex!["58f52c00c711897862f381de3a8f9be88bebfb2e37841d57215dd5aa5e51d438"].unchecked_into(),
+                    // 5FvDthRvXeUkSvNXscn8bNcHYtM9VBd78SjqZgzvxWWNoJo5
+                    hex!["aa78d4fa569bedde43d7b52b1237806b48074308eb841cea71d2b8e7e724012c"].unchecked_into(),
+                    // 5ECfNdJcAkEjt2kVB3P9BKx3ad1yGCNFFabhEYsLnUAetJud
+                    hex!["5e8a11eafe9312d6feaeff14430221ccbf4c7d85f418adfe4d7ce084ed637519"].unchecked_into(),
+                    // 5FhUeVMd3FEWfEVMddkZgoD7As8wQSekqzkKQHWYH3nL4oJN
+                    hex!["a0c04fc1d9bda258017ed3632afbd98228332f526e06ed485a6947b618ce6337"].unchecked_into(),
+                    // 5CqpnaqjVoZ8r4u3be5RuWUnAEznW7B1BXFJo7EPkXzAr3qz
+                    hex!["22695ec7b572198690d3f427df703dd53602aa2067e03c2272e38325489ab56e"].unchecked_into(),
+                    // 5G3wfLgP6GSE9kRVt5Nwi5NJ5HLznUujWo5rZQxWKpvckik7
+                    hex!["b05c28baa73f083a6ddc7dc9eb61bade0c78adc6e3c0b20d5d13a74c38eb7306"].unchecked_into(),
+                    // 5HEMPqyvfiMddVa4giN1N6mDYi2g8j4MuSzBVujmLyynvqBG
+                    hex!["e488bd0d526a384b9556f4c9200bce729863fbd9a6ec53d1812e745b136efd28"].unchecked_into(),
+                    // 5D2xyGox4gaGPczG9xK6SW46TPHByDW2JVXikDu1xNvdrvkE
+                    hex!["2ae89ed78cf10320cb59ab106cf8180f97a664d91b726bf69b5e01bd7d84060b"].unchecked_into(),
+                    // 5C5TSbMEJDFkppN4V1sCd17n25hLfXYzALM1UY3sbyBroiop
+                    hex!["0092b8488e09aaec7bca178c90a075a9fd6f2d70ecdc115dfc6fdbcb91c88876"].unchecked_into(),
+                    // 5DcP4SrNPz366T8p9XbPLZuXtjRGBrmfKavkeCnyFFjeTLJ6
+                    hex!["4464cf6dcc7b817df845816e87920fe5bffbc400334da6abd31d806ce0a34652"].unchecked_into(),
+                ],
+                vec![],
+                vec![
+                    (
+                        get_account_id_from_string("5DcrZv97CipkXni4aXcg98Nz9doT6nfs6t3THn7hhnRXTd6D"),
+                        "Interlay".as_bytes().to_vec(),
+                    ),
+                    (
+                        get_account_id_from_string("5FPBT2BVVaLveuvznZ9A1TUtDcbxK5yvvGcMTJxgFmhcWGwj"),
+                        "Band".as_bytes().to_vec(),
+                    ),
+                ],
+                id,
+                6,
```

Comment:
```suggestion
                6, // BTC block confirmations
```

Reply:
Technically would be better to have a constant at the top of the file instead of inline magic number

Reply:
Yeah I thought about this but there are so many for these configs, I've already extracted a few